lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,21 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ['https://academ.info/news/',
```

Comment:
please, check the formatting of the file, is it correct?

Reply:
fixed, thank you